clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/provider/view/DcosApplicationProvider.groovy

```diff
@@ -32,8 +32,8 @@ class DcosApplicationProvider implements ApplicationProvider {
   Set<Application> getApplications(boolean expand) {
     def relationshipFilter = expand ? RelationshipCacheFilter.include(Keys.Namespace.CLUSTERS.ns) : RelationshipCacheFilter.none()
     Collection<CacheData> applications = getAllMatchingKeyPattern(cacheView, Keys.Namespace.APPLICATIONS.ns, "${dcosCloudProvider.id}:*", relationshipFilter)
-    def secrets = getSecrets()
-    applications.collect {translate(it, secrets)}
+    //def secrets = getSecrets()
```

Comment:
This was a change I added to see if we could get away with not returning secrets on the getApplications() call, and only on the specific getApplication retrieval. This worked, so you can remove this commented out code.